clib/mininet_test_base.py

```diff
@@ -802,6 +802,15 @@ def scapy_template(packet, iface, count=1):
         return ('python3 -c \"from scapy.all import * ; sendp(%s, iface=\'%s\', count=%u)"' % (
             packet, iface, count))
 
+    def scapy_base_udp(self, mac, iface, dport, sport, count=1, dst=None):
+        if dst is None:
+            dst = 'ff:ff:ff:ff:ff:ff'
+        return self.scapy_template(
+            ('Ether(dst=\'%s\', src=\'%s\', type=%u) / '
+             'IP(src=\'0.0.0.0\', dst=\'255.255.255.255\') / '
```

Comment:
An IP src of 0.0.0.0 and dst of 255.255.255.255 doesn't really make sense here, typically that is only used for DHCP packets (port 67/68). Can we add an argument to this function that allows us to pass in the ip src and dst of the test hosts? (you could set the IP DST to the local broadcast address for the subnet if you want the packet to go to all hosts).

Reply:
Updated this to use the test host IPs